operator/cmd/run/run.go

```diff
@@ -418,6 +446,28 @@ func Run(
 		return err
 	}
 
+	var cloudExpander *pkgsecrets.CloudExpander
```

Comment:
To make the `run` method a bit less of a beast, could we construct this in the cobra command and pass the result (or `nil`) into `run`?